Formula/helm@2.rb

```diff
@@ -0,0 +1,48 @@
+class HelmAT2 < Formula
+  desc "The Kubernetes package manager"
+  homepage "https://helm.sh/"
+  url "https://github.com/helm/helm.git",
+      :tag      => "v2.16.1",
+      :revision => "bbdfe5e7803a12bbdf97e94cd847859890cf4050"
+  head "https://github.com/helm/helm.git"
```

Comment:
This line can be removed per audit log

08:42:57 ==> brew audit helm@2 --online --new-formula
08:44:28 ==> FAILED
08:44:28 Error: 1 problem in 1 formula detected
08:44:28 helm@2:
08:44:28   * Formulae should not have a `HEAD` spec

Reply:
Done, thanks @chenrui333